src/vip-framework/checks/rewardsDistributor.ts

```diff
@@ -37,6 +37,7 @@ type RewardsDistributorId =
   | "RewardsDistributor_Core_0_USDT"
   | "RewardsDistributor_Core_0_USDC"
   | "RewardsDistributor_Core_0_XVS"
+  | "RewardsDistributor_LiquidStakedETH_XVS"
```

Comment:
Shouldn't it be `RewardsDistributor_LiquidStakedETH_0_XVS`

Reply:
Added here, https://github.com/VenusProtocol/vips/pull/366/commits/bb049071bb366ebaa2c0c7256d41e37c25205125